src/shared/ui/Button/Button.tsx

```diff
@@ -0,0 +1,13 @@
+import MaterialButton, { ButtonProps as MaterialButtonProps } from '@mui/material/Button';
+
+interface ButtonProps extends MaterialButtonProps {}
+
+const Button = ({ children, ...props }: ButtonProps) => {
```

Comment:
try to avoid spread props - https://dev.to/maybebored/comment/ci01#:~:text=Child%20components%20end%20up%20accumulating,the%20origin%20can%20become%20obscured.&text=Spread%20attributes%20can%20be%20useful,HTML%20attributes%20to%20the%20DOM.
It's better to pass all needed props one by one